dist/maven-antrun/build-parallel-worlds.xml

```diff
@@ -15,6 +15,7 @@
   limitations under the License.
 -->
 <project name="maven-antrun" xmlns:ac="antlib:net.sf.antcontrib" xmlns:if="ant:if">
+    <record name="/tmp/ant-build.log" loglevel="verbose" action="start" />
```

Comment:
Is this intentional or leftover from a debugging session?  If intentional, I'd rather this was controlled via a cmdline property if possible.

Reply:
it's a leftover, forgot to commit a change to delete this line